### PR TITLE
Add JIB custom entrypoint examples on documentation

### LINKED ISSUE #27575: Improve custom entrypoint configuration documentation for JIB images
### Description

By looking at the JIB's documentation on configuring custom entry points [JIB Docs](https://quarkus.io/guides/container-image#jvm-debugging) there is not enough information on the different ways you can configure custom startup scripts for `ubi8/openjdk-11-runtime` or `ubi8/openjdk-17-runtime` images, I mean it´s quite challenging to find the right way to set the values to customize the entry point.

### Implementation ideas

For example, the documentation could be improved to provide two ways of configuring it:

1. Configuring an in-line entry point start-up command in the `application.properties` file:
```bash
quarkus.jib.jvm-entrypoint=java,-agentlib:jdwp=transport=dt_socket\\,server=y\\,suspend=n\\,address=*:5005,-jar,quarkus-run.jar
```

2. Creating a custom startup file at `src/main/jib/jboss/home/run-java.sh` with some content like below:
```bash
java \
  -Djavax.net.ssl.trustStore=/deployments/truststore \
  -Djavax.net.ssl.trustStorePassword=$TRUST_STORE_PASSWORD \
  -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005 \
  -jar quarkus-run.jar
```
and providing the custom jvm-entrypoint configuration `quarkus.jib.jvm-entrypoint=/bin/sh,run-java.sh`

### PR DESCRIPTION
**Pull request by schmittjoaopedro**
This PR regards the improvement raised at https://github.com/quarkusio/quarkus/issues/27575

### CODE REVIEW

**Review by gsmet — DISMISSED** [review]
Thanks! I will let @geoand have a look at this one. Just let's make sure we don't keep the merge commit in the PR before merging.

File: docs/src/main/asciidoc/container-image.adoc [hunk 1]
```diff
@@ -56,7 +56,34 @@ quarkus.jib.jvm-arguments=-agentlib:jdwp=transport=dt_socket\\,server=y\\,suspen
...
+
+Or a custom start-up script can be created and referenced on the properties file. This approach works better if there's a need to set application params using environment variables:
+
+.Example application.properties
+[source,properties]
+----
+quarkus.jib.jvm-entrypoint=/bin/sh,run-java.sh
+----
+
+.Example src/main/jib/home/jboss/run-java.sh
```
Comment (geoand) [inline docs/src/main/asciidoc/container-image.adoc:79]: I think a few words on why `/home/jboss` is used here would be complete the picture

**Review by geoand — COMMENTED** [review]
Very nice, thanks a lot!

I added one comment. 
Also, please rebase the PR onto the latest `main` as @gsmet suggests

File: docs/src/main/asciidoc/container-image.adoc (hunk 1 above)
Comment (schmittjoaopedro) [reply docs/src/main/asciidoc/container-image.adoc:79]: Thanks @geoand 
Done!

**Comment by geoand** [comment]
One last thing, can you please squash the commits?

**Comment by schmittjoaopedro** [comment]
> One last thing, can you please squash the commits?

Done!

**Review by geoand — APPROVED** [review]
Thanks!

**Pull request merged**